### ISSUE TITLE
How can I efficiently view my city spawns?

### ISSUE THREAD

**Issue by hdubb** [AHAAAAAAA/PokemonGo-Map#2160]
I made like 10 accounts for my city but most of them overlap.
I am using 7 steps but I would like to efficiently view the whole city.
My current setup, some overlap and some are missing.

**Comment by Sejadis** [comment]
there is a scanned location toggle in the options - should help you to find the best locations to scan

**Comment by hdubb** [comment]
Yeah, the scanned location toggle is where I put the longitude and latitude coordinates

I am figuring out how to calculate the distance between each coordinate to make entering coordinates easy and efficiently.

So far, I figured out the distance between Point A to Point B North and South at 7 steps.
Now I need to figure out the steps East to West too.

**Comment by Sejadis** [comment]
im not sure we mean the same? the toggle i mean is an overlay that shows you which areas are scanned
![scannedlocations](https://cloud.githubusercontent.com/assets/20510059/17134613/94626a1a-532b-11e6-902f-e7b9e8ba00c0.png)

**Comment by Viralpoke** [comment]
how did you do multiple locations on one map?

**Comment by hdubb** [comment]
Yes, that is what I am talking about. I want to scan my whole city so I am trying to efficiently place the scanning locations all around the area without any overlap or missing areas.

For example, some of your scanned locations are overlapping.

**Comment by hdubb** [comment]
Here is an example of what I currently have. I am trying to get my scanning location to look like the bottom picture that I drew hexagons with.
No Overlap and efficient

![pokemap](https://cloud.githubusercontent.com/assets/15079327/17135734/12de7faa-52e6-11e6-9a1f-8e765e64b820.PNG)

**Comment by Haffwa** [comment]
Hi! Not perfect, but maybe useful...
Looks like this:

![githubbild](https://cloud.githubusercontent.com/assets/17971355/17137822/8a64403a-533d-11e6-9370-6fe8dd8bd33e.png)

I used these numbers for the variation of the coordinates:
![image](https://cloud.githubusercontent.com/assets/17971355/17137897/dbeff73c-533d-11e6-95c7-a8ccd95bea91.png)

Best regards

**Comment by Plop** [comment]
https://github.com/AHAAAAAAA/PokemonGo-Map/wiki/How-to-cover-a-wider-area-with-multiple-searchers

I think this is what you're -and I was- looking for.

**Comment by hdubb** [comment]
Thanks Plop, That was exactly what I was looking for. I am calculating the coordinates manually because I couldn't figure out how to automate it and the excel script can't be edited.

**Comment by Plop** [comment]
Look for this too : https://github.com/AHAAAAAAA/PokemonGo-Map/pull/2153

**Comment by hdubb** [comment]
How can I run that code? I cant figure out how to download it.

**Comment by Plop** [comment]
@hdubb it's merged in the https://github.com/AHAAAAAAA/PokemonGo-Map/commit/fcdb919120db26c193d0f57e2b9ac6b53193032c commit. Normally in the latest commit of the develop branch too.
